### DISCUSSION TITLE
Creating component for package.

### DISCUSSION THREAD

**Discussion by ArtMin96** [livewire/livewire#2916] [1. Help]
I need to create a livewire component for my package. How can I create this with a command line?

Creating a package is very difficult for me when I first try to create a Livewire package. When I register the package ``` Livewire\Livewire::component('componentName', ComponentName::class) ``` I get the following error **Class 'Livewire\Livewire' not found**

I do not know how to create a component for a package with a command. Is there a tutorial to get started with at least a minimal setup?

**Comment by nuernbergerA** [comment]
Checkout https://github.com/tanthammar/tall-forms
